src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleLoggerProvider.cs

```diff
@@ -21,7 +21,8 @@ public class ConsoleLoggerProvider : ILoggerProvider, ISupportExternalScope
         private readonly IOptionsMonitor<ConsoleLoggerOptions> _options;
         private readonly ConcurrentDictionary<string, ConsoleLogger> _loggers;
         private ConcurrentDictionary<string, ConsoleFormatter> _formatters;
-        private readonly ConsoleLoggerProcessor _messageQueue;
+        // for testing
+        internal ConsoleLoggerProcessor MessageQueue;
```

Comment:
Do you need to make this field internal? I'm not sure your test has to actually log a message. I think you could get by with just creating a `ConsoleLoggerProvider`, calling `CreateLogger`, and looking at its `ColorBehavior`.